app/models.py

```diff
@@ -1462,10 +1464,12 @@ def validate(self, enforce_required=True, required_fields=None, max_day_rate=Non
 
     def serialize(self):
         data = self.data.copy()
+        parent_brief = self.brief.serialize()
+        parent_brief_fields = ['id', 'title', 'status', 'applicationsClosedAt']
```

Comment:
@TheDoubleK because we're always returning clarification questions when brief is serialized, which means that without JOIN we end up with N+1 (101) DB queries for each `/briefs` response from the API.

I guess the question is whether we need clarification questions in the list of briefs response at all.